File: Python/aprendizaje/Ejercicios/Impares.py
"""
7. Mostrar todos los números impares entre dos números introducidos por el usuario.
"""
n1 = int(input("Ingrese el primer numero: "))
n2 = int(input("Ingrese el segundo numero: "))

if n1 < n2:
    for i in range(n1,n2+1):
        if i & 1 != 0:
            print(i)
else:
    print("Error, el primer numero debe ser menor que el segundo")

File: Rust/basicos/Impares.rs
// 7. Mostrar todos los números impares entre dos números introducidos por el usuario.

use std::io;

fn input() -> i32 {
    loop {
        let mut input = String::new();
        let _ = io::stdin().read_line(&mut input);
        let n = input.trim();
        match n.parse::<i32>()
        {
            Ok(n) => break n,
            Err(e) => println!("Error: {}", e)
        }
    }
}

fn main() {
    println!("Ingrese un primer número: ");
    let n1 = input();
    println!("Ingrese un segundo número: ");
    let n2 = input();

    println!("######################################");
    for i in n1..=n2 {
        if i % 2 != 0 {
            println!("{}", i);
        }
    }

}
